maps/tether/tether_defines.dm

```diff
@@ -116,7 +116,7 @@
 		/area/tether/surfacebase/emergency_storage/atrium)
 
 	lateload_z_levels = list(
-		list("Tether - Misc","Tether - Ships","Tether - Underdark"), //Stock Tether lateload maps
+		list("Tether - Ships","Tether - Underdark"), //Stock Tether lateload maps
```

Comment:
`//CITADEL CHANGE - removes tether misc from lateload, it's been moved to map includes`